TestNewListenerAndURL NaCl failures

I'm trying to update the vendored pprof in the Go repository.

I've already manually cherrypicked the fix to issue #195 (I'll properly  re-vendor later when the patch gets merged), now the nacl trybots are stuck on:

```
--- FAIL: TestNewListenerAndURL (0.00s)
    --- FAIL: TestNewListenerAndURL/localhost: (0.00s)
    	webui_test.go:228: newListenerAndURL(localhost:) = listen tcp: Protocol not available
FAIL
FAIL	cmd/vendor/github.com/google/pprof/internal/driver	0.681s
2017/08/20 12:28:34 Failed: exit status 1
```

[Link to the full log](https://storage.googleapis.com/go-build-log/15b7189e/nacl-386_d005772c.log).

Looks similar to issue #144: the test fails because tcp is not available.

--- a/internal/driver/webui_test.go
+++ b/internal/driver/webui_test.go
@@ -23,6 +23,7 @@ import (
 	"net/url"
 	"os/exec"
 	"regexp"
+	"runtime"
 	"testing"
 
 	"github.com/google/pprof/internal/plugin"
@@ -193,6 +194,10 @@ func makeFakeProfile() *profile.Profile {
 }
 
 func TestNewListenerAndURL(t *testing.T) {
+	if runtime.GOOS == "nacl" {
+		t.Skip("tcp Protocol not available on Nacl")
+	}
+
 	tests := []struct {
 		hostport  string
 		wantErr   bool

--- a/internal/driver/webui_test.go
+++ b/internal/driver/webui_test.go
@@ -195,7 +195,7 @@ func makeFakeProfile() *profile.Profile {
 
 func TestNewListenerAndURL(t *testing.T) {
 	if runtime.GOOS == "nacl" {
-		t.Skip("tcp Protocol not available on Nacl")
+		t.Skip("test assumes tcp available")
 	}
 
 	tests := []struct {

